Dockerfile

```diff
@@ -1,22 +1,25 @@
-# VERSION 1.10.9
+# VERSION 1.10.15
 # AUTHOR: Matthieu "Puckel_" Roisil
+# UPGRADE BY David Wong & MelleB
 # DESCRIPTION: Basic Airflow container
 # BUILD: docker build --rm -t puckel/docker-airflow .
 # SOURCE: https://github.com/puckel/docker-airflow
 
-FROM python:3.7-slim-buster
+FROM python:3.8-slim-buster
 LABEL maintainer="Puckel_"
 
 # Never prompt the user for choices on installation/configuration of packages
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
 # Airflow
-ARG AIRFLOW_VERSION=1.10.9
+ARG AIRFLOW_VERSION=1.10.14
```

Comment:
its a version mismatch: the title says upgrade t o `1.10.15`, but here in args you specified `1.10.14`